src/components/Editor/index.js

```diff
@@ -587,18 +590,26 @@ const Editor = React.createClass({
     }
 
     if (
+      !isEnabled("editorPreview") ||
       !selectedToken ||
       !selectedFrame ||
-      !selectedExpression ||
-      !isEnabled("editorPreview")
+      !selectedExpression
     ) {
       return;
     }
 
     const token = selectedToken.textContent;
     selectedToken.classList.add("selected-token");
 
-    const value = selectedExpression.value || selectedExpression.contents.value;
+    let value = get(selectedExpression, "contents.value");
+    if (!value) {
+      const exp = this.props.getExpression(selectedExpression.value);
+      value = get(exp, "value.result");
+    }
+
+    if (!value) {
+      return;
+    }
```

Comment:
This could be a util: `getExpressionResult`

```js
let value = get(selectedExpression, "contents.value");
if (!value) {
   const exp = this.props.getExpression(selectedExpression.value);
   value = get(exp, "value.result");
 }
 if (!value) {
   return;
 }
```